### PR TITLE
Update template to correctly retrieve external IP address. Clean up sample output.

### PR DESCRIPTION
**Pull request by uluyol**
Minor cleanup of the example README. I do not get the top 2 messages shown in the examples, and PublicIPs has been deprecated. Also, add a bold notice to make the firewall note more obvious.

### CODE REVIEW

File: examples/guestbook-go/README.md [hunk 1]
```diff
@@ -169,15 +169,12 @@ redis-slave    name=redis,role=slave    name=redis,role=slave    10.0.234.24
...
 ```shell
-$ cluster/kubectl.sh get services guestbook -o template --template='{{index . "publicIPs"}}'
-current-context: "kubernetes-satnam_kubernetes"
-Running: cluster/../cluster/gce/../../_output/dockerized/bin/linux/amd64/kubectl get services guestbook -o template --template='{{.spec.publicIPs}}'
-[104.154.63.66]$
-
+$ cluster/kubectl.sh get services guestbook -o template --template='{{(index .status.loadBalancer.ingress 0).ip}}'
+23.251.156.224$
 ```
 and then visit port 3000 of that IP address e.g. `http://104.154.63.66:3000`.
```
Comment (thockin) [inline examples/guestbook-go/README.md]: This line references the IP in the old example - either make your IP be 104... or make this be 23....

**Comment by thockin** [comment]
One not then LGTM

File: examples/guestbook-go/README.md (hunk 1 above)
Comment (uluyol) [inline examples/guestbook-go/README.md]: Done.

**Comment by thockin** [comment]
LGTM

**Pull request merged**